docs/narr/project.rst

```diff
@@ -856,12 +860,19 @@ also informs Python that the directory which contains it is a *package*.
    WSGI application.  This function is meant to be called by the
    :term:`PasteDeploy` framework as a result of running ``pserve``.
 
-   Within this function, application configuration is performed.
+   :term:`PasteDeploy` configures logging from the ``.ini`` file
+   before ``main`` is called.  Within ``main`` further application
+   configuration is performed.
 
-   Line 7 opens a context manager with an instance of a :term:`Configurator`.
+   Line 7 opens a context manager with an instance of a
+   :term:`Configurator`.  This applies all configuration understood by
+   :app:`Pyramid` itself, e.g. ``pyramid.includes``, in the
+   ``[app:main]`` section of the ``.ini`` file.
 
    Line 8 adds support for Jinja2 templating bindings, allowing us to
-   specify renderers with the ``.jinja2`` extension.
+   specify renderers with the ``.jinja2`` extension.  The expectation
+   is that the extension applies what is relevant in the ``.ini`` file
+   at this point.  Consult your extension's documentation.
```

Comment:
s/extension/add-on in both new sentences. `.jinja2` is a file extension (so the usage in the original sentence is correct in that context), whereas pyramid_jinja2 is the add-on that applies ini stuff and has documentation.